client/components/tinymce/plugins/simple-payments/dialog/index.jsx

```diff
@@ -62,13 +60,21 @@ class SimplePaymentsDialog extends Component {
 		} );
 
 		this.state = {
```

Comment:
I can't do this if the state depends on `this.formStateController`. See how Babel transpiles the code that uses class properties (tested on http://babeljs.io/repl):

Source:
```js
class Test extends Component {
  state = {
    activeTab: this.props.activeTab,
    form: this.controller.form()
  };
  constructor(props) {
    super(props);
    this.controller = new Controller();
  }
}
```
Transpiled:
```js
class Test extends Component {
  constructor(props) {
    super(props);
    this.state = {
      activeTab: this.props.activeTab,
      form: this.controller.form()
    };
    this.controller = new Controller();
  }
}
```
`super` is called first, that's why I can use `this.props` when assigning to state. The rest of the constructor, however, is called only after the property assignments.